classes/oc/captcha.php

```diff
@@ -84,7 +84,7 @@ public static function check($name = '', $ajax = FALSE)
         //d(Session::instance()->get('captcha_'.$name));
         //d(Session::instance()->get('captcha_'.$name) == strtolower(core::post('captcha')));
 
-        if (core::config('advertisement.captcha') == FALSE) // Captchas are disabled
+        if (core::config('advertisement.captcha') == FALSE AND core::config('general.captcha') == FALSE) // Captchas are disabled
```

Comment:
On OE `core::config('advertisement.captcha')` is `NULL` therefore `(core::config('advertisement.captcha') == FALSE) ? FALSE : TRUE` is always `FALSE`.

With just `if (core::config('general.captcha') == FALSE)` then on OC it won't work since it’s always `FALSE` because there is no ‘general.captcha’ config.